examples/java/web-service/pom.xml

```diff
@@ -0,0 +1,85 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.mycompany.app</groupId>
+  <artifactId>my-app</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-SNAPSHOT</version>
+  <name>my-app</name>
+  <url>http://maven.apache.org</url>
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
+      <scope>test</scope>
+    </dependency>
+    <!-- https://mvnrepository.com/artifact/org.cdk8s/cdk8s -->
+    <dependency>
+      <groupId>org.cdk8s</groupId>
+      <artifactId>cdk8s</artifactId>
+      <version>0.24.0</version>
+    </dependency>
+    <dependency>
+      <groupId>software.constructs</groupId>
+      <artifactId>constructs</artifactId>
+      <version>2.0.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>29.0-jre</version>
+  </dependency>  
+  <dependency>
+    <groupId>software.amazon.jsii</groupId>
+    <artifactId>jsii-runtime</artifactId>
+    <version>1.5.0</version>
+</dependency>
+  </dependencies>
+  <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+  <build>
+    <sourceDirectory>src/main/java</sourceDirectory>
+    <testSourceDirectory>src/test/java</testSourceDirectory>
+    <resources>
+      <resource>
+        <directory>imports/src/main/k8s/main/java</directory>
+      </resource>
+      <resource>
+        <directory>imports/src/main/k8s/main/resources</directory>
+      </resource>
```

Comment:
Something feels wrong here. Is there any way we can change how import works so we don't need to add this for every new import?

Reply:
I can look into it... wasn't seeming to work with just `imports/` unfortunately. Might be a Maven thing too

Reply:
Let's try to find a way that pom.xml file does not need updates when importing new APIs.

Reply:
Simplified it so there's only one place you need to add for every new import. Seems like this is how maven expects it to be:

https://maven.apache.org/plugins/maven-resources-plugin/examples/resource-directory.html

Reply:
Why can't we just add all the imports under the same source tree, so there is no need to update the pom file at all for new imports?

Reply:
Ahhh, now I understand. I'll go ahead and do that!